types/readable-stream/package.json

```
@@ -20,6 +20,10 @@
         {
             "name": "markdreyer",
             "githubUsername": "markdreyer"
+        },
+        {
+            "name": "mcollina",
+            "githubUsername": "mcollin"
```

```suggestion
            "githubUsername": "mcollina"
```

This looks like a typo.